plugins/versionpress/src/Cli/vp-composer.php

```diff
@@ -20,6 +22,17 @@
  */
 class VPComposerCommand extends WP_CLI_Command
 {
+
+    private $pluginsThemesTransient = 'vp_composer_plugins_themes';
```

Comment:
is `Transient` a term for naming transient options in WordPress world, or there can be also other transient objects/values/etc.

My note is if the variable is named correctly.